content/en/docs/concepts/overview/working-with-objects/names.md

```diff
@@ -26,7 +28,7 @@ See the [identifiers design doc](https://git.k8s.io/community/contributors/desig
 
 By convention, the names of Kubernetes resources should be up to maximum length of 253 characters and consist of lower case alphanumeric characters, `-`, and `.`, but certain resources have more specific restrictions.
 
```

Comment:
Maybe shorten "should be up to maximum length of 253 characters".

Should have a maximum of 253 characters